Resize don't resample pixel aspect ratio change

lucienfostier opened the pull request:
Improvements:
-------------------

- Resize(#2007):
  - Ignore the pixel aspect ratio to test if we need to resample the image when resizing.

johnhaddon commented:
Thanks Lucien - the intention here looks good. I have a few questions though :

- Should we really be using `outPlug()->formatPlug()`? Can't we just continue to use `formatPlug()` as before?
- Could you also tweak it so that we don't call `getValue()` for each plug twice, but instead store the results in a const variable?
- For the rare case of non-zero format origins, won't this mean the resize does nothing when one origin is zero and the other isn't? Should we instead compare the display windows?

lucienfostier commented:
Hey John,

I've addressed your comment for this pull request.
To give you a bit of context, we are currently working on switching to Gaffer to ingest client's plate and this is an important piece when dealing with anamorphic material.

Cheers

johnhaddon commented:
Great - thanks Lucien!
